src/js/themes/hpe.js

```diff
@@ -851,7 +851,7 @@ export const hpe = deepFreeze({
       container: {
         gap: 'xsmall',
       },
-      icon: <CircleAlert size="small" />,
+      icon: () => <CircleAlert size="small" />,
```

Comment:
@ericsoderberghp Are these changes also necessary? Just asking because no issues have been flagged by them prior

Reply:
I'm just aligning all of the icons to work the same way. I don't know that it really matters. But, it seemed like this might be slightly lighter weight.